backend/src/openarchiefbeheer/destruction/models.py

```diff
@@ -510,6 +510,37 @@ def __str__(self):
         return f"Case review for {self.destruction_list} ({self.destruction_list_item})"
 
 
+class DestructionListCoReview(models.Model):
+    """
+    Model linking a co reviewer (User) to a destruction list.
+    Actual (temporary) review items are stored using selection API
```

Comment:
This comment confused me :sweat_smile: while the code makes it clear what this is, so I think it can be removed :thinking: